### PR TITLE
Configurable zabbix server port and database port

### PR DESCRIPTION
**Pull request by vincepii**

### CODE REVIEW

File: tasks/postgresql.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 - name: "PostgreSQL | Create database"
   postgresql_db: name={{ server_dbname }}
+                 port={{ server_dbport }}
```
Comment (lhoss) [inline tasks/postgresql.yml]: `server_dbport` not yet in the `defaults/main.yml`

Comment (vincepii) [inline tasks/postgresql.yml]: Added default value to the variable, thanks for spotting it.

**Comment by dj-wasabi** [comment]
Can you also update the readme.md with this? It seems the default value is set for the PostGreSQL port. People with MySQL databases will need to update their configuration for this.

**Comment by dj-wasabi** [comment]
p.s. I'll try to fix the CI test today and when fixed, i'll merge the PR.

**Comment by vincepii** [comment]
Many thanks! I added some information in the README file.

**Comment by dj-wasabi** [comment]
Thanks!

**Pull request merged**